.github/workflows/dev-fe-e2e.yaml

```diff
@@ -16,6 +16,13 @@ jobs:
   e2e:
     runs-on: ubuntu-latest
     steps:
+      - name: Switch docker-daemon data directory to /mnt
+        run: |
+          sudo systemctl stop docker
+          echo '{ "exec-opts": ["native.cgroupdriver=cgroupfs"], "cgroup-parent": "/actions_job", "data-root": "/mnt/docker-data" }' | sudo tee /etc/docker/daemon.json
+          sudo mkdir /mnt/docker-data
+          sudo systemctl start docker
+
```

Comment:
I suppose this is some sort of speed up, could you please elaborate more on why it's needed?

Reply:
Ok, I see the same step was added in the scope of this PR https://github.com/percona/everest/pull/1074/files with a comment, so now things got more clear. Looks like previously some tests were failing because of running out of space and this change allows to get more space.